build.gradle

```diff
@@ -16,7 +16,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.1'
+        classpath 'com.android.tools.build:gradle:3.2.0'
```

Comment:
change gradle version to 3.1.4 or remove it here, since the one declared at app level takes effect